src/math/fields/fields.zig

```diff
@@ -601,6 +604,28 @@ pub fn Field(comptime F: type, comptime modulo: u256) type {
             );
         }
 
+        pub fn fromSignedInt(comptime T: type, signed: T) Self {
+            if (signed < 0) {
+                return Self.fromInt(u256, STARKNET_PRIME - @as(u256, @intCast(@abs(signed))));
+            }
+
+            return Self.fromInt(T, signed);
+        }
+        // converting felt to abs value with sign
+        pub fn toSignedInt(self: Self) struct { positive: bool, abs: u256 } {
```

Comment:
Can we add unit tests in the dedicated file in order to confirm the validity of the method?